content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -1,296 +1,425 @@
 
 
 #### Create a new seed asset list
-Create a new see asset list that contains only your level assets. 
 
-2. Create a new seed list file by clicking **Create a new Seed List file** in the **Seed List file** panel under the **Seeds** tab
-3. Select your new seed list file from the list. 
-4. Click **+ Add Asset** in the **Product Assets** panel. This opens the **Add Seed Asset** dialog. 
-5. In the list of platforms, enable **pc**.
-6. Click **Browse...** and navigate to the `levels` folder and choose **Add Seed**. 
-7. Verify that your seed list has the level assets in the **Product Assets** panel. 
+1. In the Asset Bundler, on the **Seeds** tab, in the **Seed List file** panel, click **Create a new Seed List file**. For this example, name the file `GameSeedList`.
+
+2. Select the `GameSeedList` file from the list.
+
+3. In the **Product Assets** panel, click **+ Add Asset**, which opens the **Add Seed Asset** dialog.
+
+4. In the list of platforms, select the **pc** check box.
+
+5. Click **Browse...**, which opens File Explorer.
 
-#### Generate an asset list 
+6. In File Explorer, navigate to the `levels` folder, and then choose **Add Seed**.
 
-1. Select your new seed list file by enabling the check box. 
-2. Click **Generate Asset Lists**. This will open the **Generate Asset List files** dialog. 
-3. In the list of platforms, select **pc** by enabling the check box.  
-4. Click **Browse...** to open the File Explorer. 
-5. Enter a name for your asset list and save. In this example, we'll use the name `game.assetlist`. 
-5. Navigate to the **Asset Lists** tab to verify the assets in `game.assetlist`. The assets are listed under the **Asset List** panel. 
+7. In the Asset Bundler, in the **Product Assets** panel, verify that your seed list has the level assets.
+
+#### Generate an asset list
+
+1. In the Asset Bundler, select your new seed list file from the list.
+
+2. Click **Generate Asset Lists**, which opens the **Generate Asset List files** dialog.
+
+3. In the list of platforms, select the **pc** check box.
+
+4. Click **Browse...**, which opens File Explorer.
+
+5. In File Explorer, enter a name for your asset list, and then click **Save**. For this example, use the name `game.assetlist`.
+
+6. In the Asset Bundler, navigate to the **Asset Lists** tab to verify the assets in `game.assetlist`. The assets are listed in the **Asset List** panel.
 
 #### Bundle your assets
 
-1. Select your `game.assetlist` in the** Asset List Files** panel under the **Asset Lists** tab.
+1. In the Asset Bundler, on the **Asset Lists** tab, in the **Asset List Files** panel, select your asset list (`game.assetlist`).
 
-2. Click **Generate Bundle**. This opens the **Generate Bundles** dialog. 
+2. Click **Generate Bundle**, which opens the **Generate Bundles** dialog.
 
-3. Click **Browse...** to open the File Explorer. 
+3. Click **Browse...**, which opens the File Explorer.
```

Comment:
```suggestion
1. Click **Browse...**, which opens File Explorer.
```
Does this actually open Windows' own File Explorer tool? If so, drop "the", as it's a proper noun. If not, don't use "File Explorer." Instead, call it something generic (", which opens a file browser.").